src/components/Follow/FollowList.tsx

```diff
@@ -0,0 +1,137 @@
+import TextButton, { StyledButtonText } from 'components/core/Button/TextButton';
+import colors from 'components/core/colors';
+import Markdown from 'components/core/Markdown/Markdown';
+import Spacer from 'components/core/Spacer/Spacer';
+import { BaseM, TitleS } from 'components/core/Text/Text';
+import { useIsMobileOrMobileLargeWindowWidth } from 'hooks/useWindowSize';
+import { useState } from 'react';
+import { graphql, useFragment } from 'react-relay';
+import styled from 'styled-components';
+
+type Props = {
+  userRef: any;
+};
+
+const getFirstLine = (text: string) => (text ? text.split('\n')[0] : '');
```

Comment:
nit: even if `text` is empty, it'll return an empty string

<img width="211" alt="image" src="https://user-images.githubusercontent.com/12162433/170088603-3eb6700d-1bf7-4efd-b7ef-b5107da1346a.png">

```js
const getFirstLine = (text: string) => text.split('\n')[0]
```

but if the text param is _optional_ `(text?: string)` then your solution makes sense!